frontend/language/src/nb.json

```diff
@@ -1696,6 +1704,8 @@
   "ux_editor.file_upload_component.display_mode": "Type",
   "ux_editor.file_upload_component.settings": "Innstillinger for filopplastingskomponent",
   "ux_editor.file_upload_component.valid_file_endings": "Innstillinger for filopplastingskomponent",
+  "ux_editor.formLayout.warning_duplicates.text1": "Du har den samme ID-en på flere komponenter: ",
```

Comment:
I believe we can rename ```text'``` and ```text2``` to something more understandable. What about: 
- ```ux_editor.formLayout.warning_duplicates```, and 
- ```ux_editor.formLayout.warning_duplicates_cannot_publish```
? 😄